deeplake/core/chunk_engine.py

```diff
@@ -964,7 +967,15 @@ def _extend(self, samples, progressbar, pg_callback=None, update_commit_diff=Tru
             return
         if len(samples) == 0:
             return
-        samples, verified_samples = self._sanitize_samples(samples)
+        samples, verified_samples = self._sanitize_samples(
+            samples, pg_callback=pg_callback
+        )
+        if (
+            isinstance(self, deeplake.core.linked_chunk_engine.LinkedChunkEngine)
```

Comment:
if you can have a method that you can override in LinkedChunkEngine, it'll be better